js/web/lib/backend-onnxjs.ts

```diff
@@ -3,10 +3,40 @@
 
 /* eslint-disable import/no-internal-modules */
 import {Backend, InferenceSession, SessionHandler} from 'onnxruntime-common';
+
+import {Logger} from './onnxjs/instrument';
 import {Session} from './onnxjs/session';
 import {OnnxjsSessionHandler} from './onnxjs/session-handler';
 
 class OnnxjsBackend implements Backend {
+  protected getOnnxjsSessionConfig(options?: InferenceSession.SessionOptions): Session.Config {
```

Comment:
ALWAYS use normal functions instead of class methods unless you need to access class members. In a module organized code base we try to minimize usage of class.

Reply:
this has been taken out, will have another pr with logger related changes.